036-BGP/README.md

```diff
@@ -0,0 +1,59 @@
+# What The Hack - Using BGP for Hybrid Connectivity
+
+## Introduction
+
+In this WTH exercise students will explore how to use BGP in a complex environment between an on-premises network and two Azure regions. Both active/passive and active/active Azure VPN Gateways will be used in Azure, and Cisco CSR 1000v will be used to simulate onprem devices.
+
+The challenges will show real-world scenarios that have been observed in customers deploying their applications to Azure
+
+The estimated duration time for this hack is 1.5-2 days.
+
+## Learning Objectives
+
+After completing this WTH, participants will be familiar with these concepts, amongst others:
+
+- BGP
+- Differences between eBGP and iBGP
+- Understanding Autonomous Systems
+- Using AS-path prepending to influence routing
+- Explore BGP route manipulation capabilities
+- BGP communities
+
+## Before you start
+
+Please read these instructions carefully:
+
+- Setting expectations: all of the challenges in this FastHack would take around 2 days to complete. Please do not expect to finish all of the exercises in a shorter event
```

Comment:
Remove reference to FastHack and grep -r the rest of the files just in case. :-)

Reply:
I would tweak the wording slightly to indicate:

"While this hack was designed with enough content for a 2-day event, you will still get value if you complete only some of the challenges in a shorter event."

Reply:
Abut the fasthack references, it seems that you dont have the last version, I already removed the references in the last review. Hopefully it will be fixed after the next commit.

Reply:
@jrzyshr what you said